backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -182,7 +190,10 @@ public void updatePatient(
     this.birthDate = birthDate;
     this.address = address;
     this.role = role;
-    this.email = email;
+    this.emails = emails;
+    if (emails != null && !emails.isEmpty()) {
+      this.email = emails.get(0);
```

Comment:
move this logic to the getEmail() method. we dont need to maintain an email field, but we need to keep the getEmail() for compatibility.